includes/Discovery.php

```diff
@@ -41,7 +41,7 @@ class Discovery {
 	 * @return void
 	 */
 	public function init() {
-		add_action( 'web_stories_story_head', [ $this, 'print_metadata' ], 1 );
+		add_action( 'web_stories_story_head', [ $this, 'print_metadata' ] );
```

Comment:
Isn't this number required for the title to render?

Reply:
This was how it was before.

Reply:
There's no reason why the priority would need to be like that. The title is displayed just fine this way.